README.md

```diff
@@ -75,6 +79,12 @@ We provide two pre-trained models:
 
   [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3370486.svg)](https://doi.org/10.5281/zenodo.3370486)
 
+Furthermore, we provide a model for speech enhancement trained by [Sony Corporation](link)
```

Comment:
link ?

Reply:
@TE-StefanUhlich do we have a link to add or just zenodo?
Can you prepare a short markdown paragraph for the [sigsep open-unmix page](https://github.com/sigsep/website/blob/master/content/open-unmix/README.md) ?

Reply:
@faroit The link is the following: https://zenodo.org/record/3786908

What do you think of the following paragraph?

* __`umxse`__ is in contrast a speech enhancement model which does a separation into `speech` and `noise`. It is trained on the 16kHz version of the [VoiceBank+DEMAND corpus](https://datashare.is.ed.ac.uk/handle/10283/1942) which is a common dataset for speech enhancement and, therefore, this model can be compared to other speech enhancement approaches.

  [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3786908.svg)](https://zenodo.org/record/3786908)

Reply:
Looks good. I will add this to the sigsep site. Can we add the Sony logo at the bottom of the page?